doc/mkdoc.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+sofa_src=/home/tgaugry/sofa/src/master
+build_dir=/home/tgaugry/sofa/build/master/Release
+sofapython3_src=$(realpath ..)
+include_dirs="-I/usr/lib64/llvm5.0/lib/clang/5.0.1/include -I${build_dir}/include -I${sofa_src}/SofaKernel/framework -I${sofapython3_src}/binding -I/usr/include/python3.7m"
```

Comment:
Bheu...c'est quoi ça :)

Reply:
That's actually in the TODO in the first message :D I wanted to be sure we take this approach before committing to cmake madness